src/components/InformationPanes.js

```diff
@@ -31,17 +33,18 @@ function debugBtn(onClick, type, className, tooltip) {
   );
 }
 
-const RightSidebar = React.createClass({
+const InformationPanes = React.createClass({
```

Comment:
I don't have a particular affinity for the name either, something more descriptive would be good. 👍